### ISSUE TITLE
\ does not work on sparse complex systems

### ISSUE THREAD

**Issue by ghost** [JuliaLang/LinearAlgebra.jl#38]
``` julia
A = sprand(100,100,0.1); A = A'*A;
A = A + im*speye(100);
b = randn(100);
x = A\b
Error code -4 from numeric factorization
```

This type of systems are very important to many problems in electrical engineering and geoscience

[jiahao: edit for code formatting]

**Comment by andreasnoack** [comment]
It works in my machine with

```
julia> versioninfo()
Julia Version 0.2.0-rc2+129
Commit ba5ef5d 2013-11-02 00:47:05 UTC
Platform Info:
  System: Darwin (x86_64-apple-darwin13.0.0)
  WORD_SIZE: 64
  BLAS: libopenblas (USE64BITINT DYNAMIC_ARCH NO_AFFINITY)
  LAPACK: libopenblas
  LIBM: libopenlibm
```

What do you get from `versioninfo()`?

**Comment by ghost** [comment]
I have
versioninfo()
Julia Version 0.2.0-rc1
Commit 040f3ab 2013-10-15 05:58:39 UTC
Platform Info:
  System: Darwin (x86_64-apple-darwin12.5.0)
  WORD_SIZE: 64
  BLAS: libgfortblas
  LAPACK: liblapack
  LIBM: libopenlibm

**Comment by jiahao** [comment]
Duplicate of JuliaLang/julia#4523 (fixed in JuliaLang/julia#4536).

I can confirm that this is reproducible on 0.2.0-rc1 (040f3ab) and also that it is fixed in the latest version. Please file a new issue if the problem persists after upgrading.
